system_tests/bigquery.py

```diff
@@ -33,6 +38,14 @@ def _make_dataset_name(prefix):
     return '%s%s' % (prefix, unique_resource_id())
 
 
+def _load_json_schema(filename='bigquery_test_schema.json'):
+    from google.cloud.bigquery.table import _parse_schema_resource
```

Comment:
I would add a newline after the import.

I was looking at adding this to lint through pycodestyle but I haven't proposed the change yet.